R/html-tweak.R

```diff
@@ -124,17 +124,23 @@ tweak_rmarkdown_html <- function(html, input_path) {
   invisible()
 }
 
-tweak_homepage_html <- function(html, strip_header = FALSE) {
+tweak_homepage_html <- function(html, strip_header = FALSE, sidebar = TRUE) {
   badges <- badges_extract(html)
-  if (length(badges) > 0) {
+
+  if (!sidebar) {
+    sidebar_html <- html %>% xml2::xml_find_first(".//div[@id='pkgdown-sidebar']")
```

Comment:
Hmmm, I don't like the idea of completely removing this div. Why can't we just leave it empty?